charts/kubernetes-external-secrets/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: kubernetes-external-secrets
-version: 7.1.0
+version: 7.2.0
```

keep as is or release scripts probably go sad / mismatching
```suggestion
version: 7.1.0
```